flink-runtime/src/main/java/org/apache/flink/runtime/util/config/memory/jobmanager/JobManagerFlinkMemoryUtils.java

```diff
@@ -75,12 +75,12 @@ private static JobManagerFlinkMemory createJobManagerFlinkMemory(
 			MemorySize jvmHeap,
 			MemorySize offHeapMemory) {
 		verifyJvmHeapSize(jvmHeap);
-		verifyJobStoreCacheSize(config, offHeapMemory);
+		verifyJobStoreCacheSize(config, jvmHeap);
 		return new JobManagerFlinkMemory(jvmHeap, offHeapMemory);
 	}
 
 	private static void verifyJvmHeapSize(MemorySize jvmHeapSize) {
-		if (jvmHeapSize.compareTo(JobManagerOptions.MIN_JVM_HEAP_SIZE) < 1) {
+		if (jvmHeapSize.compareTo(JobManagerOptions.MIN_JVM_HEAP_SIZE) < 0) {
```

Comment:
why is this changed? I think zero is also ok, meaning that `jvmHeapSize` can be exactly the `MIN_JVM_HEAP_SIZE`.

Reply:
+1 for using 0; otherwise you can run into the following error which will always be perceived as an oversight:
`The configured or derived JVM heap memory size (128.000mb (134217728 bytes)) is less than its recommended minimum value (128 mb)`

Reply:
@zentol I agree with you. @azagrebin The log tells the users the comparation result is "less than".

Reply:
True, sorry for confusion